toolkit/smart-contracts/plutus-data/src/candidate_keys.rs

```diff
@@ -1,8 +1,16 @@
+//! Functions related to PlutusData decoding and encoding of candidates keys
 use cardano_serialization_lib::{PlutusData, PlutusList};
 
 pub(crate) type KeyTypeId = [u8; 4];
 pub(crate) type KeyTypeIdAndBytes = (KeyTypeId, Vec<u8>);
 
+/// Well known polkadot-sdk id of AURA key type
+pub const AURA_TYPE_ID: &KeyTypeId = b"aura";
+/// Well known polkadot-sdk id of Grandpa key type
+pub const GRANDPA_TYPE_ID: &KeyTypeId = b"gran";
```

Comment:
just in case, [here](https://docs.rs/sp-core/latest/sp_core/crypto/key_types/index.html) is a registry of this identifiers. Probably later for our purposes we should create similar one